enterprise/config/buildbuddy.local.yaml

```diff
@@ -13,8 +13,58 @@ storage:
     root_directory: /tmp/buildbuddy_enterprise
   enable_chunked_event_logs: true
 cache:
+  # Sample flow if migration disk -> pebble
+
+  # Begin double writing, and still return data from disk
   disk:
     root_directory: /tmp/buildbuddy-enterprise-cache
+    migration_src: true
+  pebble:
+    root_directory: /tmp/buildbuddy-enterprise-cache
+    migration_dest: true
+
+  # Copy all data from disk -> pebble
+  disk:
+    root_directory: /tmp/buildbuddy-enterprise-cache
+    migration_src: true
+  pebble:
+    root_directory: /tmp/buildbuddy-enterprise-cache
+    migration_dest: true
+  migration_cache:
+    migrate_and_exit: true
```

Comment:
Super minor idea - I wonder if there's some nice way we could keep all of the migration stuff in the migration block. That way you wouldn't have to add this (and corresponding flags) to each cache implementation. It also would mean it would be harder to misconfigure by setting multiple caches as src/dest.

Something like:
```
migration:
  source: disk
  dest: pebble
  double_read_percentage: 0.1
```

Then you could switch on these in the migration code and set the right src/dest.

Also wondering how we would handle a pebble => pebble migration - say with a different filesystem layout.

Just some thoughts / ideas. I like the overall design!

Reply:
Good call out! Updated the initialization mechanism if you want to take another look @siggisim @tylerwilliams @vadimberezniker @bduffany